### ISSUE TITLE
Responsiveness 🙌

### ISSUE THREAD

**Issue by tonylomax** [fac-17/BGIP#23]
Pretty great, on iPhone5 the table gets cut off a little. 

 
![Screen Shot 2019-08-23 at 10 19 33](https://user-images.githubusercontent.com/44951051/63581678-8393af80-c58f-11e9-92d9-9934d4af661f.png)

**Comment by bobbysebolao** [comment]
Yep, 100% agree, it's great that you're designing mobile-first!!!
